advisor_listener/advisor_listener.py

```diff
@@ -225,12 +226,15 @@ def update_cves(cur, to_update, rh_account_id):
 def main():
     # pylint: disable=too-many-branches, too-many-statements
     """Application entrypoint"""
-    start_http_server(int(PROMETHEUS_PORT))
     init_logging()
     ensure_minimal_schema_version()
```

Comment:
The `/healthz` endpoint has to be available before `ensure_minimal_schema_version()` is called. This function can be waiting for undefined time but the container will be restarted while waiting by OpenShift if doesn't respond to health checks.

Reply:
nice catch, should be resolved now